charts/beyla/values.yaml

```diff
@@ -186,6 +186,21 @@ config:
     attributes:
       kubernetes:
         enable: true
+      select:
+        beyla_network_flow_bytes:
+          include:
+            - 'k8s.src.owner.type'
+            - 'k8s.dst.owner.type'
+            - 'direction'
```

Comment:
We actually need the cluster name, owner name and namespaces too, but I saw in the documentation they are already on by default, so the list here is just the additional attributes Asserts uses.

Alternatively belya can also make them available by default, so we don't need to have set them here.